1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,13 @@ let basketballTeam = {
 */
 
 // write code here
-
+const topPlayersList = basketballTeam.topPlayers;
+console.log(topPlayersList.sort().join("\n"));
 
```

Comment:
here we wanted you to show player names separately, at the moment with your code we have their names in one long string, can you change your code? :)

Reply:
@ellietms Thanks for the review 😊
Here is my new approches :
`basketballTeam.topPlayers.sort().forEach((player) => console.log(player));`
or
`basketballTeam.topPlayers.sort().map((player) => console.log(player));`

Reply:
Thank you Elli for reviewing.
I have now update it using array 😊